infra/auth0/terraform.tf

```
@@ -0,0 +1,65 @@
+terraform {
+  backend "s3" {
+    bucket = "monoweb-terraform"
+    key    = "auth0.tfstate"
+    region = "eu-north-1"
+  }
+
+  required_version = "~> 1.7.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.33"
+    }
+    doppler = {
+      source  = "DopplerHQ/doppler"
+      version = "~> 1.3.0"
+    }
+    auth0 = {
+      source  = "auth0/auth0"
+      version = "~> 1.1.0"
+    }
+  }
+}
+
+variable "DOPPLER_TOKEN_ALL" {
```

Should stick with terraform namving convention despite it being used as an environment variable
```suggestion
variable "doppler_token_all" {
```

---

Doppler does not support non-uppercasing the env-variable, so I suggest we don't